### PR TITLE
{geo}[foss/2021b] QGIS v3.28.1, GRASS v8.2.0, Qt5Webkit v5.212.0-alpha4, wxPython v4.2.0, attrdict3 v2.0.2, librttopo v1.1.0, minizip v1.1,  w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

Depends on: ~~https://github.com/easybuilders/easybuild-easyblocks/pull/2845~~

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 61 out of 69 (15 easyconfigs in total)
himem03.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/bbbf9e7a6a9fcc8192fb82912bfd3722 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 1 out of 8 (15 easyconfigs in total)
himem03.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/022d87e4f98310a020a81fd14a2fa28a for a full test report.

**Comment by akesandgren** [comment]
I knew I forgot something... qscintilla.py coming up.

**Comment by akesandgren** [comment]
@jfgrimm See https://github.com/easybuilders/easybuild-easyblocks/pull/2845

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2845
**FAILED**
Build succeeded for 1 out of 7 (15 easyconfigs in total)
himem03.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/02989d378d5a341db71809ab62a3a5f4 for a full test report.

**Comment by jfgrimm** [comment]
`QScintilla` works for me with the easyblock update, but I'm now failing to build `libwpe`:

```
228 [ 38%] Building CXX object CMakeFiles/wpe.dir/src/pasteboard-generic.cpp.o                                              
229 /opt/apps/easybuild/software/GCCcore/11.2.0/bin/g++ -DWPE_COMPILATION -DWPE_ENABLE_XKB=1 -Dwpe_EXPORTS -I/dev/shm/jfg508/libwpe/1.13.3/GCCcore-11.2.0/libwpe-1.13.3/include -I/dev/shm/jfg    508/libwpe/1.13.3/GCCcore-11.2.0/libwpe-1.13.3/src -O2 -ftree-vectorize -march=native -fno-math-errno -fPIC -fno-exceptions -fno-rtti -Wall -O3 -DNDEBUG -fPIC -fvisibility=hidden -std=gn    u++11 -MD -MT CMakeFiles/wpe.dir/src/pasteboard-generic.cpp.o -MF CMakeFiles/wpe.dir/src/pasteboard-generic.cpp.o.d -o CMakeFiles/wpe.dir/src/pasteboard-generic.cpp.o -c /dev/shm/jfg508/    libwpe/1.13.3/GCCcore-11.2.0/libwpe-1.13.3/src/pasteboard-generic.cpp
230 In file included from /usr/include/string.h:633,                                                                        
231                  from /dev/shm/jfg508/libwpe/1.13.3/GCCcore-11.2.0/libwpe-1.13.3/src/pasteboard.c:32:                   
232 /usr/include/bits/string2.h:1305:33: error: attempt to use poisoned "calloc"                                            
233  1305 |                      ? (char *) calloc ((size_t) 1, (size_t) 1)               \                                 
234       |                                 ^                                                                               
235 /usr/include/bits/string2.h:1307:53: error: attempt to use poisoned "malloc"                                            
236  1307 |                           char *__retval = (char *) malloc (__len);           \                                 
237       |                                                     ^                                                           
238 /usr/include/bits/string2.h:1325:33: error: attempt to use poisoned "calloc"                                            
239  1325 |                      ? (char *) calloc ((size_t) 1, (size_t) 1)               \                                 
240       |                                 ^                                                                               
241 /usr/include/bits/string2.h:1331:47: error: attempt to use poisoned "malloc"                                            
242  1331 |                           __retval = (char *) malloc (__len);                 \                                 
243       |                                               ^                                                                 
244 make[2]: *** [CMakeFiles/wpe.dir/src/pasteboard.c.o] Error 1
```

**Comment by akesandgren** [comment]
fortunately for me libwpe is not my problem :-)

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by jfgrimm** [comment]
`libwpe` issue resolved in https://github.com/easybuilders/easybuild-easyconfigs/pull/17001 :)

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2845
**SUCCESS**
Build succeeded for 17 out of 17 (15 easyconfigs in total)
node064.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/afafe472d9c47e758808e332ed0b58e0 for a full test report.

File: easybuild/easyconfigs/g/GRASS/GRASS-8.2.0-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+configopts += '--with-zstd --with-zstd-libs=$EBROOTZSTD/lib --with-zstd-includes=$EBROOTZSTD/include '
+
+postinstallcmds = [
+    'ln -s grass%(version_major)s%(version_minor)s %(installdir)s/grass%(version_major)s',
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["."]
+}
```
Comment (jfgrimm) [inline easybuild/easyconfigs/g/GRASS/GRASS-8.2.0-foss-2021b.eb:100]: @akesandgren this needs fleshing out

Comment (akesandgren) [reply easybuild/easyconfigs/g/GRASS/GRASS-8.2.0-foss-2021b.eb:100]: Ah did I forget that...

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node064.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ac16277a395b12f59e6e993a0e9e438b for a full test report.

**Review by jfgrimm — APPROVED** [review]
lgtm

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/b31b8698366e094b0168909655f0792f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 15 out of 15 (15 easyconfigs in total)
node3130.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ee29c1616039390a80acc687a52e64d5 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**